packages/ncids-css/scss/components/nci-header/internal/_search.scss

```diff
@@ -1,96 +1,89 @@
 // SEARCH STYLES
 .nci-header--extended {
-  .nci-header-nav__secondary {
-    &.search-focused {
-      .nci-header-mobilenav__open-btn {
-        display: none;
-        @include at-media('mobile-lg') {
-          display: block;
-        }
-        @include at-media('desktop') {
-          display: none;
-        }
-      }
-    }
+	.nci-header-nav__secondary {
+		&.search-focused {
+			.nci-header-mobilenav__open-btn {
+				@include u-display('none');
+				@include at-media('mobile-lg') {
+					@include u-display('block');
+				}
+				@include at-media('desktop') {
+					@include u-display('none');
+				}
+			}
+		}
 
-    .nci-header-search {
+    // form wrapping sws input
+		.nci-header-search {
 			flex-grow: 1;
-      margin: 0;
+			margin: 0;
 			white-space: nowrap;
 			position: relative;
+			@include u-maxw(100%);
```

Comment:
I believe this should also be `@include u-maxw('full')`

However, I don't see errors in the build log, which has me concerned. Oh, I see it is because we don't import any of the components into our _all.scss. :(

So yeah, this is an issue and it is not showing in the NCIDS build logs because of other issues.